### DISCUSSION TITLE
Custom domain is not working.  ERR_NAME_NOT_RESOLVED

### DISCUSSION THREAD

**Discussion by m90khan** [vercel/community#7084] [Help]
### Summary

I have attached my custom domain and the status is all green. however, I am unable to site via my custom domain. It does not just work on network but also on other networks. I tested with mobile data on different mobile networks.

The website error is ERR_NAME_NOT_RESOLVED

Domain: kr8v.com

Registrar: NameCheap

kr8v.vercel.app works.

**custom domain works when using VPN. tested all places.**

`╭─    ~ ──────────────────────────────────────────────── ✔  14:44:53  ─╮
╰─ date                                                                      ─╯
Di  4 Jun 2024 14:44:57 PKT

╭─    ~ ──────────────────────────────────────────────── ✔  14:44:57  ─╮
╰─ curl https://kr8v.com                                                     ─╯
curl: (6) Could not resolve host: kr8v.com

╭─    ~ ────────────────────────────────────── 6 ✘  30s   14:45:37  ─╮
╰─ curl https://ipinfo.io                                                    ─╯
{
  "ip": "124.29.253.33",
  "city": "Bhalwal",
  "region": "Punjab",
  "country": "PK",
  "loc": "32.2658,72.8981",
  "org": "AS9541 Cyber Internet Services (Pvt) Ltd.",
  "postal": "40410",
  "timezone": "Asia/Karachi",
  "readme": "https://ipinfo.io/missingauth"
}%

╭─    ~ ──────────────────────────────────────────────── ✔  14:49:40  ─╮
╰─ dig vercel.com                                                            ─╯

; <<>> DiG 9.10.6 <<>> vercel.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 5597
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;vercel.com.			IN	A

;; ANSWER SECTION:
vercel.com.		262	IN	A	76.76.21.22
vercel.com.		262	IN	A	76.76.21.9

;; Query time: 56 msec
;; SERVER: 192.168.1.1#53(192.168.1.1)
;; WHEN: Tue Jun 04 14:50:14 PKT 2024
;; MSG SIZE  rcvd: 71

╭─    ~ ──────────────────────────────────────────────── ✔  14:50:14  ─╮
╰─ dig kr8v.com                                                              ─╯

; <<>> DiG 9.10.6 <<>> kr8v.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 27837
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;kr8v.com.			IN	A

;; Query time: 9 msec
;; SERVER: 192.168.1.1#53(192.168.1.1)
;; WHEN: Tue Jun 04 14:50:52 PKT 2024
;; MSG SIZE  rcvd: 37

╭─    ~ ──────────────────────────────────────── ✔  10s   14:50:52  ─╮
╰─ ping -c 4 cname.vercel-dns.com                                            ─╯
PING cname.vercel-dns.com (76.76.21.98): 56 data bytes
64 bytes from 76.76.21.98: icmp_seq=0 ttl=246 time=33.570 ms
64 bytes from 76.76.21.98: icmp_seq=1 ttl=246 time=36.335 ms
64 bytes from 76.76.21.98: icmp_seq=2 ttl=246 time=37.587 ms
64 bytes from 76.76.21.98: icmp_seq=3 ttl=246 time=36.528 ms

--- cname.vercel-dns.com ping statistics ---
4 packets transmitted, 4 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 33.570/36.005/37.587/1.484 ms

╭─    ~ ───────────────────────────────────────── ✔  3s   14:52:00  ─╮
╰─ dig kr8v.com @8.8.8.8                                                     ─╯

; <<>> DiG 9.10.6 <<>> kr8v.com @8.8.8.8
;; global options: +cmd
;; connection timed out; no servers could be reached

╭─    ~ ────────────────────────────────────── 9 ✘  15s   14:52:43  ─╮
╰─ curl -sv https://kr8v.com                                                 ─╯
* Could not resolve host: kr8v.com
* Closing connection 0`

![namecheap](https://github.com/vercel/community/assets/6104751/a3b98787-7de5-467c-985f-913f9f0b9e0b)

![vercel](https://github.com/vercel/community/assets/6104751/64ad15b6-9d82-4970-ba5b-9567ba45e370)

### Steps to Reproduce

There are no steps to reproduce the error.

**Comment by amyegan** [comment]
I see the domain is configured with [the appropriate A record](https://www.whatsmydns.net/#A/kr8v.com), and I was able to visit the site just now without error. 

![image](https://github.com/vercel/community/assets/8247105/0116eb17-f8da-4eb5-8b8d-f3e1e84dcdd2)

Since it works for others and works on VPN, it could be a local issue. In that case, you many need to [clear the DNS cache in your browser](https://www.makeuseof.com/chrome-edge-firefox-safari-opera-browser-clear-dns-cache/) or [flush DNS cache on your device](https://www.hostinger.com/tutorials/how-to-flush-dns) to get it working.

**Reply by m90khan** [reply]
Thanks for the help.
It is not working in Pakistan. 
I have tested with many devices and different browsers and networks. so cache is not the issue here.

`╰─ curl -sv https://kr8v.com    ─╯
* Could not resolve host: kr8v.com
* Closing connection 0`
